### ISSUE TITLE
Update CLI 1.7.4 to 6 fails with fsevents error with Node 10 on OSX

### ISSUE THREAD

**Issue by chriscurnow** [angular/angular-cli#10898]
### Versions
```

node: v10.0.0
npm: v5.6.0
macOs High Sierra


```

### Repro steps

* Attempting to update to Angular 6 from CLI 1.7.4 following update instructions
* `ng update @angular/cli` does not update CLI
* fall back to cli update instructions in this repo README
* run `npm install --save-dev @angular/cli@latest`

### Observed behavior
```

First lines of stacktrace after `npm install --save-dev @angular/cli@latest`:

> fsevents@1.1.2 install /Projects/4cast/4castPro/4castPro-front/node_modules/fsevents
> node install

node-pre-gyp ERR! Tried to download(404): https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.1.2/fse-v1.1.2-node-v64-darwin-x64.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for fsevents@1.1.2 and node@10.0.0 (node-v64 ABI) (falling back to source compile with node-gyp) 
```

### Desired behavior
Should install correctly or update instructions should be updated for node 10.

### Mention any other details that might be useful (optional)
When the update instructions say "ensure you are using Node 8 or later" I presumed node 10 would be fine but this appears to be the problem.

The fix is relatively easy, once you workout the problem.

Before attempting to install CLI, run

`npm install fsevents@1.2.4 --save`

**Comment by filipesilva** [comment]
I'm not sure what we can do here. We have tested on Node 10 before but never ran into this. It looks like a version `fsevents` somewhere in node_modules either doesn't support node 10, or just failed to download that one time.

**Comment by chriscurnow** [comment]
@filipesilva Thanks.

I mainly raised an issue here so anyone else experiencing it could get a heads up on how they might address it 😄

**Comment by filipesilva** [comment]
I think that was a good thing to do. As a maintainer I find it very important to have these 'heads up' issues around for other people searching the issue tracker. 

They shouldn't be open as there isn't any fix/action item in this project proper, but that they were brought up and discussed is already a valuable artifact for the future.

**Comment by thehashrocket** [comment]
yep, it does the same on my mac with node 10.7.0. It's been throwing errors for the past 3 days. tried force clearing the cache. still no love.

**Comment by OctaneInteractive** [comment]
I'm having this problem with Node v10.8.0

I see `fsevents` listed in both: "yarn.lock" (`fsevents "^1.0.0"`) and: "package-lock.json" (`"fsevents": "1.1.2"`) files, but not in their regular counterparts.

**Comment by thekip** [comment]
For someone who also has this errors (i'm using Yarn)

>  yarn why fsevents

```
=> Found "fsevents@1.0.15"
info Has been hoisted to "fsevents"
info Reasons this module exists
   - Hoisted from "gulp-watch#chokidar#fsevents"
   - Hoisted from "@angular#compiler-cli#chokidar#fsevents"
=> Found "chokidar#fsevents@1.2.4"
info This module exists because "chokidar" depends on it.
info Disk size without dependencies: "4.36MB"
info Disk size with unique dependencies: "7.06MB"
info Disk size with transitive dependencies: "17.41MB"
info Number of shared dependencies: 83
```

Well, this tells us that `gulp-watch` and `@angular/compiler-cli` caused old version of fsevents to be in node_modules. 

So than do 

> yarn remove @angular/compiler-cli && yarn add --dev @angular/compiler-cli

and the same for any other packages which stacked on old version (gulp, gulp-watch and etc).

The root cause of problem that old version of fsevents stored in lock file, and you need to update "dependecy of dependecy". 
Instead of changing .lock file manually better way is delete and reinstall dependecy.

**Comment by grosch** [comment]
Creating a brand new angular project with node 10 will exhibit the same message, and then you've gotta sit through a build of fsevents.
